src/bin/lpython.cpp

```diff
@@ -1049,8 +1135,8 @@ extern "C" { // using extern "C" to prevent function name mangling
 EMSCRIPTEN_KEEPALIVE char* emit_ast_from_source(char *input) {
     INITIALIZE_VARS;
     lm.init_simple(input);
-    LFortran::Result<LFortran::LPython::AST::Module_t*> ast = LFortran::parse(al, input, diagnostics);
-    out = diagnostics.render(input, lm, compiler_options);
+    LFortran::Result<LFortran::LPython::AST::Module_t*> ast = LFortran::parse(al, input, 0, diagnostics);
+    out = diagnostics.render(lm, compiler_options);
```

Comment:
Only the WASM has to be fixed.
I have a doubt, how does the input value is being passed here?